src/vm/callingconvention.h

```diff
@@ -417,11 +417,10 @@ class ArgIteratorTemplate : public ARGITERATOR_BASE
 
 #ifdef _TARGET_AMD64_
 #ifdef UNIX_AMD64_ABI
-        PORTABILITY_ASSERT("ArgIteratorTemplate::IsVarArgPassedByRef");                
         return FALSE;
-#else // UNIX_AMD64_ABI
+#else // !UNIX_AMD64_ABI
```

Comment:
The comment was correct - the convention in the VM is to always use the condition from the #ifdef, not the negation.

Reply:
Ok will address.